src/Raven.Studio/typescript/viewmodels/database/tasks/EditReplicationHubInfoHub.tsx

```diff
@@ -0,0 +1,38 @@
+﻿import AboutViewFloating, { AccordionItemWrapper } from "components/common/AboutView";
+import AccordionLicenseNotIncluded from "components/common/AccordionLicenseNotIncluded";
+import { licenseSelectors } from "components/common/shell/licenseSlice";
+import { useAppSelector } from "components/store";
+import React from "react";
+import {Icon} from "components/common/Icon";
+
+export function EditReplicationHubInfoHub() {
+    const isEnterpriseOrDeveloper = useAppSelector(licenseSelectors.isEnterpriseOrDeveloper());
+
+    return (
+        <AboutViewFloating>
```

Comment:
It should be open if view is limited. Example (line 63 and 84):
https://github.com/kalczur/ravendb/blob/d34aa5c18e6a25e13be2bb85b5b2a3fe987026b5/src/Raven.Studio/typescript/components/pages/database/indexes/cleanup/IndexCleanup.tsx#L63-L94

Reply:
Same for rest of info hubs

Reply:
Thanks, done